website/content/en/docs/AWS/constraints.md

```diff
@@ -11,9 +11,7 @@ This section covers parameters of the AWS Cloud Provider.
 [Review these fields in the code.](https://github.com/awslabs/karpenter/blob/main/pkg/cloudprovider/aws/apis/v1alpha1/provider.go#L33)
 
 ### InstanceProfile
-An `InstanceProfile` is a way to pass a single IAM role to an EC2 instance.
-
-It is required, and specified by name. A suitable `KarpenterNodeRole` is created in the getting started guide.
+An `InstanceProfile` is a way to pass a single IAM role to an EC2 instance. This can be specified to override the default profile specified during installation.

```

Comment:
note to self: add link or example giving some indication of what "specified during installation" means -- perhaps helm chart readme?